packages/harpguru-core/src/hooks/use-scaled-menu-label-protrusion/use-scaled-menu-label-protrusion.ts

```diff
@@ -1,10 +1,11 @@
-import { useSizes } from '../use-sizes'
+import { useGlobal } from 'reactn'
+
 import { menuStashedScale } from '../../constants'
 
 export const useScaledMenuLabelProtrusion = (): number => {
   const {
-    dynamicSizes: { labelProtrusion: unscaledLabelProtrusion },
-  } = useSizes()
+    [0]: { labelProtrusion: unscaledLabelProtrusion },
+  } = useGlobal('dynamicSizes')
```

Comment:
Similar point to https://github.com/js-jslog/harpguru/pull/137#discussion_r765155671

Reply:
Done